### ISSUE TITLE
Add Targeted IDS for CSS Customization

### ISSUE THREAD

**Issue by SaltysKing** [wisp-gg/frontend#318]
### Describe the feature and why you would like to see it.

Add Targeted IDS for CSS Customization, within each HTML element has a different ID so we can have further advanced CSS customization.

Right now the most you can do is remove portions of the panel, and its really hard to do much like extend the size of the text editor. Making it more friendly for less advanced CSS designers as well.

### Describe the solution you'd like.

making each HTML element have a different ID, for example, this could be done automatically using JS and just adding it to each element loaded in.

**Comment by victiondev** [comment]
Is there anything in particular you need id's for, as its hard from our perspective to guess what people want to customise - we're happy to add id's to whatever you need to make it easier to target

**Comment by SaltysKing** [comment]
Yes things like the background, titles for each module, the file and text editor module.
